### PR TITLE
Eliminate cached include paths (#105)

### PR DESCRIPTION
**Pull request by nist-anka**
Whenever possible use only target_include_directories in CMakeLists.txt.
Needs building in fresh directory or removal of CMakeCache.txt

### CODE REVIEW

File: IbisVTK/CMakeLists.txt [hunk 1]
```diff
@@ -1,11 +1,12 @@
-SET( IBIS_VTK_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/vtkExtensions ${CMAKE_CURRENT_SOURCE_DIR}/vtkMNI ${CMAKE_CURRENT_SOURCE_DIR}/vtkQt  )
-SET( IBIS_VTK_INCLUDE_DIR ${IBIS_VTK_INCLUDE_DIR} PARENT_SCOPE  )
-
 #================================
 # Define include dir for
 # dependent projects
 #================================
 
 add_subdirectory( vtkExtensions )
+target_include_directories( vtkExtensions PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/vtkExtensions
```
Comment (drouin-simon) [inline IbisVTK/CMakeLists.txt]: Why is this not in vtkExtensions/CMakeLists.txt ?

File: IbisVTK/vtkQt/CMakeLists.txt [hunk 2]
```diff
@@ -35,32 +34,3 @@ if( IBIS_INSTALL_LIBS_AND_HEADERS )
     INSTALL_TARGETS(/lib vtkQt)
     INSTALL_FILES(/include FILES ${IBIS_VTK_QT_HDR} )
 endif( IBIS_INSTALL_LIBS_AND_HEADERS )
-
```
Comment (drouin-simon) [inline IbisVTK/vtkQt/CMakeLists.txt]: Why are we getting rid of Plugins for QtCreator ?

File: IbisVTK/CMakeLists.txt (hunk 1 above)
Comment (nist-anka) [reply IbisVTK/CMakeLists.txt]: That was to replace lines

SET( IBIS_VTK_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/vtkExtensions ${CMAKE_CURRENT_SOURCE_DIR}/vtkMNI ${CMAKE_CURRENT_SOURCE_DIR}/vtkQt  )
SET( IBIS_VTK_INCLUDE_DIR ${IBIS_VTK_INCLUDE_DIR} PARENT_SCOPE  )

in the original  IbisVTK/CMakeLists.

File: IbisVTK/vtkQt/CMakeLists.txt (hunk 2 above)
Comment (nist-anka) [reply IbisVTK/vtkQt/CMakeLists.txt]: Do you mean QT_EXTENSIONS_INCLUDE_DIR?
It is not defined or used  anywhere.

**Pull request merged**